### PR TITLE
Add flag for disabling port switching

### PR DESCRIPTION
**Pull request by jespertheend**
https://github.com/zeit/serve/issues/365

### CODE REVIEW

File: bin/serve.js [hunk 1]
```diff
@@ -370,6 +370,7 @@ const loadConfig = async (cwd, entry, args) => {
 			'--no-etag': Boolean,
 			'--symlinks': Boolean,
 			'--cors': Boolean,
+			'--no-port-switching': Boolean,
```
Comment (leerob) [inline bin/serve.js:373]: Could you add documentation for this?

Comment (jespertheend) [reply bin/serve.js:375]: I've added it to --help. Would you like to have it added somewhere else as well?

Comment (leerob) [reply bin/serve.js:375]: True, that will probably suffice. Thank you.

**Review by leerob — APPROVED** [review]

**Comment by JamesHenry** [comment]
@leerob Sorry to be commented on a merged PR, but it seems a shame to open an issue just for this:

It looks like there hasn't been a release of the library for a year now - please could you cut one so that we can easily make use of this change via NPM?

Thanks!

**Comment by JamesHenry** [comment]
Hmm additionally @jespertheend was it your intention that this _doesn't_ exit with code 1?

Having installed it via the git commit and run it, it always seems to hit `INFO: Gracefully shutting down. Please wait...` and exits with code 0 which is unintuitive I think

```
ERROR: Failed to serve: Error: listen EADDRINUSE: address already in use :::4400
    at Server.setupListenHandle [as _listen2] (net.js:1318:16)
    at listenInCluster (net.js:1366:12)
    at Server.listen (net.js:1452:7)
    at startEndpoint (/Users/james/.../node_modules/serve/bin/serve.js:220:19)
    at /Users/james/.../node_modules/serve/bin/serve.js:445:3

INFO: Gracefully shutting down. Please wait...
```

**Comment by jespertheend** [comment]
Ah interesting, I hadn’t even thought about that to be honest. Non-zero seems more intuitive yeah. I was just reusing the shutting down code that was already in place I believe. So it should behave the same as when an error other than EADDRINUSE is thrown.

**Comment by leerob** [comment]
This has now been released 👍

**Comment by JamesHenry** [comment]
Thanks @leerob, just installed it instead of the commit reference and everything looks good!

**Pull request merged**